src/org/labkey/test/components/ui/pipeline/ImportsPage.java

```diff
@@ -0,0 +1,96 @@
+package org.labkey.test.components.ui.pipeline;
+
+import org.apache.tika.utils.StringUtils;
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.components.ui.grids.QueryGrid;
+import org.labkey.test.pages.LabKeyPage;
+import org.labkey.test.util.URLBuilder;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.openqa.selenium.WebElement;
+
+public class ImportsPage extends LabKeyPage<LabKeyPage<?>.ElementCache>
+{
+
+    public ImportsPage(WebDriverWrapper driver)
+    {
+        super(driver);
+    }
+
+
+    @Override
+    protected void waitForPage()
+    {
+        waitFor(()-> {
+                    try
+                    {
+                        return elementCache().pageHeader().isDisplayed() &&
+                                elementCache().pipelineJobsGrid().isLoaded();
+                    }
+                    catch(NoSuchElementException | StaleElementReferenceException nse)
+                    {
+                        return false;
+                    }
+                },
+                "The 'Background Imports' page did not load in time.",
+                2_500);
+    }
+
+    public static ImportsPage beginAt(WebDriverWrapper webDriverWrapper)
+    {
+        return beginAt(webDriverWrapper, webDriverWrapper.getCurrentContainerPath());
+    }
+
```

Comment:
I've gotten feedback from Trey that beginAt should not have this overloaded. We should always have the container path provided (like below).
I suspect this is the implementation in LKSM, but I have been trying to update the various page objects when I come across them.

Reply:
I don't quite understand why this overload is a bad idea, but I've removed it by essentially inlining what the method does.